### PR TITLE
Add sex symbol to audio sources list

### PR DESCRIPTION
**Pull request by MarvNC**
It adds a biological sex symbol to the text that is displayed. If you think the string might look better by rearranging things that would be fine.

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -94,7 +94,15 @@ def word(self, w):
...
             username = re.search(r"Pronunciation by([^(]+)\(",i.get_text(strip=True)).group(1).strip()
-            audio_sources.append({"name":f"Forvo ({username})","url":url})
+            gender = re.search(r"\((Male|Female)",i.get_text(strip=True))
+            gender = gender.group(1).strip() if gender is not None else ''
+            genderSymbols = {
+                'Male': '♂',
+                'Female': '♀'
+            }
+            genderSymbol = genderSymbols[gender]
+            genderSymbol = genderSymbol if genderSymbol is not None else ''
```
Comment (cyphar) [inline __init__.py]:
```suggestion
            genderSymbol = genderSymbols.get(gender, "")
```

File: __init__.py [hunk 2]
```diff
@@ -94,7 +94,15 @@ def word(self, w):
             # Capture the username of the user
             # Some users have deleted accounts which is why can't just parse it from the <a> tag
             username = re.search(r"Pronunciation by([^(]+)\(",i.get_text(strip=True)).group(1).strip()
-            audio_sources.append({"name":f"Forvo ({username})","url":url})
+            gender = re.search(r"\((Male|Female)",i.get_text(strip=True))
+            gender = gender.group(1).strip() if gender is not None else ''
```
Comment (cyphar) [inline __init__.py]:
```suggestion
            gender = re.search(r"\((Male|Female)",i.get_text(strip=True)).group(1).strip()
```

File: __init__.py [hunk 3]
```diff
@@ -220,4 +228,4 @@ def do_GET(self):
     httpd = http.server.ThreadingHTTPServer(('localhost', 8770), ForvoHandler)
     server_thread = threading.Thread(target=httpd.serve_forever)
     server_thread.daemon = True
-    server_thread.start()
+    server_thread.start()
```
Comment (cyphar) [inline __init__.py:229]: No need to remove the last newline.

**Review by cyphar — CHANGES_REQUESTED** [review]

**Comment by MarvNC** [comment]
Thank you @cyphar!

**Comment by jamesnicolas** [comment]
Going to merge this and make some config changes before deploying

File: __init__.py (hunk 2 above)
Comment (cyphar) [reply __init__.py]: My bad, this was actually wrong to suggest. Fixed in #12.

**Pull request merged**